README.md

```diff
@@ -1,25 +1,29 @@
-# Getting Started with Create React App
+# Coveo Headless Starter Project
+
+This project serves as a starting point for the Coveo Headless Tutorial (**coming soon!**).
+
+## Getting Started with Create React App
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+### Available Scripts
 
 In the project directory, you can run:
 
```

Comment:
If we give readers the commands, I'd also mention "npm install" somewhere :)
Maybe you could simply start this sentence with:

After running `npm install` in the project directory, you can run: